vaadinfrontend/src/main/resources/messages.properties

```diff
@@ -1,4 +1,4 @@
 GENERAL=Ooooooops! Sorry! -> Please contact support@qbic.zendesk.com.
 INVALID_PROJECT_TITLE=The project title {1} is invalid. -> A valid project title is not empty and is shorter than {0} characters.
-INVALID_EXPERIMENTAL_DESIGN=The experimental design description is too long. -> A description of an experimental design must contain at most {0} characters.
+INVALID_EXPERIMENTAL_DESIGN=The experimental design description is invalid -> A vaild description of an experimental design must not be empty and contain at most {0} characters.
```

Comment:
```suggestion
INVALID_EXPERIMENTAL_DESIGN=The experimental design description is invalid -> A vaild description of an experimental design must not be empty and contain at least the number of samples.
```

Reply:
@jenniferboedker I think this is the description of the experimental design ergo meaning the textual explanation what is to happen with the samples. 
Currently we inform the user if that description is over a specific amount of characters. 
Do we change the definition of the experimental design to only account for the number of samples?

Reply:
? what do you mean with that @jenniferboedker ?